bpf/init.sh

```diff
@@ -413,9 +424,8 @@ else
 	fi
 fi
 
+# BPF host program attachment
 case "${MODE}" in
```

Comment:
remove this case as it it for all modes

Reply:
Yep, will do once I have policy properly working.